src/spikeinterface/sorters/external/kilosort2.py

```diff
@@ -162,16 +166,17 @@ def _get_specific_options(cls, ops, params):
         ops["Th"] = projection_threshold
 
         # how important is the amplitude penalty (like in Kilosort1, 0 means not used, 10 is average, 50 is a lot)
-        ops["lam"] = 10.0
+        ops["lam"] = params["lam"]
 
         # splitting a cluster at the end requires at least this much isolation for each sub-cluster (max = 1)
         ops["AUCsplit"] = params["AUCsplit"]
 
         # minimum spike rate (Hz), if a cluster falls below this for too long it gets removed
         ops["minFR"] = params["minFR"]
 
+        momentum = [float(mom) for mom in params["momentum"]]
```

Comment:
The same was done above for `threshold_projection`, but I don't know exactly why so I did this just to be safe.